ast/parser.go

```diff
@@ -849,6 +849,21 @@ func (p *Parser) parseLiteral() (expr *Expr) {
 					p.hint("`import future.keywords.every` for `every x in xs { ... }` expressions")
 				}
 			}
+			// If we can't find `every or some` still check if import future.keywords.in
+			if term, ok := expr.Terms.(*Term); ok && (!Var("every").Equal(term.Value) && !Var("some").Equal(term.Value)) {
```

Comment:
When would `term.Value` be a _var_ "every" or "some"? Ah, if the imports aren't there... hmm this is probably OK 🤔